flavors/documentation/Dockerfile

```diff
@@ -182,64 +231,59 @@ RUN ML_THIRD_PARTY_DIR="/third-party/shellcheck" \
     && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 
 ENV PATH="/root/.cargo/bin:${PATH}"
-RUN cargo install shellcheck-sarif
+RUN cargo install shellcheck-sarif \
 
 # shfmt installation
-COPY --from=shfmt /bin/shfmt /usr/bin/
+# Managed with COPY --from=shfmt /bin/shfmt /usr/bin/
 
 # hadolint installation
-COPY --from=hadolint /bin/hadolint /usr/bin/hadolint
+# Managed with COPY --from=hadolint /bin/hadolint /usr/bin/hadolint
 
 # editorconfig-checker installation
-COPY --from=editorconfig-checker /usr/bin/ec /usr/bin/editorconfig-checker
+# Managed with COPY --from=editorconfig-checker /usr/bin/ec /usr/bin/editorconfig-checker
 
 # dotenv-linter installation
-RUN wget -q -O - https://raw.githubusercontent.com/dotenv-linter/dotenv-linter/master/install.sh | sh -s
+    && wget -q -O - https://raw.githubusercontent.com/dotenv-linter/dotenv-linter/master/install.sh | sh -s \
 
 # ktlint installation
-RUN curl --retry 5 --retry-delay 5 -sSLO https://github.com/pinterest/ktlint/releases/latest/download/ktlint && \
+    && curl --retry 5 --retry-delay 5 -sSLO https://github.com/pinterest/ktlint/releases/latest/download/ktlint && \
     chmod a+x ktlint && \
-    mv "ktlint" /usr/bin/
-
+    mv "ktlint" /usr/bin/ \
 
 # kubeval installation
-RUN ML_THIRD_PARTY_DIR="/third-party/kubeval" \
+    && ML_THIRD_PARTY_DIR="/third-party/kubeval" \
     && mkdir -p ${ML_THIRD_PARTY_DIR} \
     && wget -P ${ML_THIRD_PARTY_DIR} -q https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz \
     && tar xf ${ML_THIRD_PARTY_DIR}/kubeval-linux-amd64.tar.gz --directory ${ML_THIRD_PARTY_DIR} \
     && mv ${ML_THIRD_PARTY_DIR}/kubeval /usr/local/bin \
-    && find ${ML_THIRD_PARTY_DIR} -type f -not -name 'LICENSE*' -delete -o -type d -empty -delete
-
+    && find ${ML_THIRD_PARTY_DIR} -type f -not -name 'LICENSE*' -delete -o -type d -empty -delete \
 
 # kubeconform installation
-RUN ML_THIRD_PARTY_DIR="/third-party/kubeconform" \
```

Comment:
Here, the `ML_THIRD_PARTY_DIR` is an env variable set for the current line or group of commands (I can't remember the name of this shell syntax).

Once merged all together in a big `RUN`, how does setting env variables inline multiple times work?
This same variable was set multiple times for different tools, is there some side effects once in a big `RUN`?

Reply:
setting several times the same env variable is useless but harmless :)

Reply:
But each time they are different values, is the answer still the same?

Reply:
They are used only by the lines after their assignment so that's ok :) (if not, CI tests would have failed ^^ )